io/plugins/eu.esdihumboldt.hale.io.jdbc/src/eu/esdihumboldt/hale/io/jdbc/constraints/DatabaseTable.java

```diff
@@ -36,14 +36,30 @@
 	// Database support.
 	private final boolean useQuote;
 
+	private final boolean isTable;
+
+	/**
+	 * Create a default constraint.
+	 * 
+	 */
+	public DatabaseTable() {
+		this.schemaName = null;
+		this.tableName = null;
+		this.useQuote = false;
+		this.isTable = false;
+	}
+
 	/**
 	 * Create a default constraint. The default table name is the given type
 	 * local name.
 	 * 
 	 * @param type the type definition the table is associated to
 	 */
 	public DatabaseTable(TypeDefinition type) {
```

Comment:
This constructor should be removed. It does not make sense to create constraints that state that a type is not a table and at the same time provide a table name.